.github/mergify.yml

```diff
@@ -24,19 +24,29 @@ queue_rules:
 
 # NOTE: in case of flaky tests you above checks will fail
 # the PR will be removed from the queue.
-# This retries the checks, as soon as they become green,
-# the PR will be pusehd to the queu again
+# Once the PR is ready to be merged the flaky tests will
```

Comment:
I would put a limit in retries ...

Reply:
I'll investigate If there is a sane way to do this. Currently I did not find any nice solutions